### DISCUSSION TITLE
Simple Statamic Site on Raspberry Pi with no Internet

### DISCUSSION THREAD

**Discussion by grovberg** [statamic/cms#8682] [General]
Short Version:
Any reason I couldn't run a simple Statamic site locally from a Raspberry Pi with no internet access?

Long Version:
I've dabbled in Statamic for some time but never built anything that runs in production with it before so I thought I should check my assumptions here. We're working on what is effectively an interactive museum exhibit that will allow patrons to interact with an oral history collection. For a variety of reasons this exhibit needs to run without access to the internet, but it seems like I could run a Statamic site locally from a Raspberry Pi hooked up to a touchscreen. I especially like the idea that I could run a private copy of the entire site elsewhere to allow the content experts to update the content, and then just manually move the new entries to the Pi to update the exhibit.

I did some light Googling about running Statamic from a Raspberry Pi in general and didn't find much, but I can't see any reason it shouldn't work. Anyone know any hidden gotchas I might into in a situation like that?

Thanks.

**Comment by duncanmcclean** [comment]
I *think* it should be fine to do. You'll likely need internet access when you set it up to download dependencies, etc but beyond that you should be fine!
